.azure-pipelines.yml

```diff
@@ -0,0 +1,22 @@
+jobs:
+  - job: Windows
+    strategy:
+      matrix:
+        MSVC_2015:
+          MUMBLE_QT: qt5
+          MUMBLE_HOST: x86_64-pc-windows-msvc
+        MSVC_2015_NO_PCH:
+          MUMBLE_QT: qt5
+          MUMBLE_HOST: x86_64-pc-windows-msvc
+          MUMBLE_NO_PCH: 1
+    steps:
+    - script: git submodule --quiet update --init --recursive
+      displayName: 'Fetch submodules'
+    - powershell: |
+        Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))
```

Comment:
The script is signed. Can we check that signature?

Maybe this is a bit too careful, but downloading and running remote ps1 scripts can be argued to be questionable? So we should verify them, or commit them locally? (in the buildenv libs we download and checksum check our dep packages for example)

Reply:
That is meant to be a temporary solution, as 7-Zip is installed on the Visual Studio 2017 image: https://github.com/Microsoft/azure-pipelines-image-generation/pull/302

We will build a new environment with Visual Studio 2017 once 1.3 is out.

Reply:
Now that you mention it, isn't this whole thing supposed to replace appveyor for mac builds? But with this script we build only windows with MSVC??

Reply:
AppVeyor doesn't support macOS, we use Travis CI for that.

Reply:
I thought it was the other way around. Okay.